scripts/install_pytorch.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mamba env update --file $(dirname "$0")/../conda_recipe/pytorch_deps_environment.yml
```

Comment:
Did you still want to use mamba or conda?

Reply:
If I actually have to run it, I'd prefer mamba. Can change it if conda is still preferred. @arthurp